.github/workflows/backend-cd.yml

```
@@ -0,0 +1,86 @@
+      - name: 서버 다시 올리기
+        run: |
+          JAR_FILE=$(find ./artifacts -name "*.jar" | head -n 1)
+          echo "Starting server with $JAR_FILE"
+          RUNNER_TRACKING_ID=""
+          nohup java -jar "$JAR_FILE" \
+          -Dspring.profiles.active="${{ secrets.DEV_PROFILE }}" \
+          --server.port=80 \
+          --DB_URL="${{ secrets.DEV_DB_URL }}" \
+          --DB_USERNAME="${{ secrets.DEV_DB_USERNAME }}" \
+          --DB_PASSWORD="${{ secrets.DEV_DB_PASSWORD }}" \
+          > app.log 2>&1 &
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)